### DISCUSSION TITLE
OPENDTU

### DISCUSSION THREAD

**Discussion by schauriman** [tbnobody/OpenDTU#1928] [Q&A]
Ich habe openDTU geflasht , komme über den Access POint auch in die Einstellungen, kann IP Sub net  usw. alles eintragen ist dann auch in der Übersicht zu sehen,  nur meine SSID ist dort nicht drin und eine Verbindung zu meinem Netz bekomme ich auch nicht.
Hat jemand das Problem schon mal gehabt und kennt eventuell eine Lösung dafür ?

**Comment by MetaChuh** [comment]
@schauriman

bitte gib in den netzwerkeinstellungen deine wlan ssid und dein wlan passwort ein, wie im untenstehenden screenshot abgebildet.

sollte dies bei dir trotzdem nicht funktionieren, überprüfe ob in deinem router ein 2.4ghz kanal über 11 eingestellt ist.
wenn ja, dann probiere einen kanal zwischen 1 und 11 aus und reboote die opendtu zur sicherheit danach neu.

<img width="1502" alt="Bildschirmfoto 2024-04-20 um 18 43 25" src="https://github.com/tbnobody/OpenDTU/assets/45773305/0fd2eeb5-c601-4736-b236-0ca3e55aa443">

greetings,
metachuh

**Comment by schauriman** [comment]
Hallo,
genau so habe ich es eingegeben, ich habe mit und ohne DHCP versucht. Mein Funkkanal ist die 10
Er bekommt keine Verbindung und in den Onfo Daten ist die SSID nicht drin.
Ist die Anzahl der Stellen im Feld SSID begrenzt ?

**Reply by MetaChuh** [reply]
das wäre interessant auszuprobieren.
wie viele stellen hat deine ssid ?

und kannst du mal probieren ein gäste-wlan oder mobilen handy hotspot mit kurzer ssid zu machen, und kontrollieren, ob sich der esp32 mit diesem wlan/hotspot verbinden kann ?

**Comment by schauriman** [comment]
ich habe das mit dem Gäste WLAN versucht, hat funktioniert
dann habe ich mein eigentliches WLAN wieder eingetragen und 
es geeeeeeeht,  keine Ahnung warum 
ich habe zwar noch keine Verbindung zum Wechselrichter aber immerhin schon mal ein Netz

**Reply by MetaChuh** [reply]
super, jeder teilerfolg ist gut 👍

beim inverter ist wichtig, dass wir wissen welcher inverter es genau ist, und ob das passende funkmodul angeschlossen und korrekt konfiguriert ist.

am besten im ersten schritt einen screenshot von der `info > system` seite ganz unten machen.

beispiel screenshot für hoymiles hm inverter (nicht für hms/hmt inverter):

<img width="1358" alt="Bildschirmfoto 2024-04-21 um 14 03 07" src="https://github.com/tbnobody/OpenDTU/assets/45773305/d086af33-9d51-41e8-aaf9-b9506dec9ec7">

**Comment by schauriman** [comment]
Wechselrichter ist HMS-800W-2T
also CMT2300
ist dran aber steht als nicht konfiguriert drin

**Reply by ghost** [reply]
Hi,
du hast ein falsches Pinmapping geladen, bzw. deine tatsächliche Konfiguraton nicht unter Einstellungen - Hardware - Anschlusseinstellungen - Ausgewältes Profil ausgewählt. 
Du benötigst eine passende pin_mapping.json, oder  machst dier selber eine für dein [Device Profile](https://www.opendtu.solar/firmware/device_profiles/#structure-of-the-json-file) (Beispiele [hier](https://github.com/tbnobody/OpenDTU/tree/master/docs/DeviceProfiles)) und wählst deine Konfig nach dem hochladen (Konfigurationsverwaltung) aus.
(Hinweis aus eigener schmerzlicher Erfahrung: Beim Hochladen auch Pin Mapping als Ziel auswählen.)

**Reply by MetaChuh** [reply]
@schauriman 

oje, der hms-800w-2t wird leider nicht mit opendtu und auch nicht mit ahoydtu oder onbattery funktionieren.
dieser inverter kann nur wifi und hat kein cmt2300a

siehe auch zb: https://github.com/tbnobody/OpenDTU/discussions/1449

**Reply by MetaChuh** [reply]
[`Inverter Kompatibilitätsliste`](https://www.opendtu.solar/hardware/inverter_overview/)

**Reply by ghost** [reply]
Das ist bitter. 
Da hilft auch kein korrigiertes Pinmapping.

@schauriman 
Der Hoymiles HMS-800-2T  geht mit CMT2300A .
Sprich: kein "W" nach der 800!

@MetaChuh
Danke, ist mir nicht aufgefallen.
Mein Tipp mag zwar für den CMT hilfreich sein,  für eine komplette Funktion bei **schauriman** ist der Tipp leider unbrauchbar.

**Comment by schauriman** [comment]
ja toll, so oft nachgesehen und immer überlesen
man sollte eben genauer hin schauen
schade
hat jemand sonst eine Idee wie ich die Daten zum MQTT bekomme  ??

**Reply by MetaChuh** [reply]
wenn du issue #1449 und die weiterführenden links gelesen hast, landest du unter anderem auf https://github.com/DennisOSRM/hms-mqtt-publisher

**Comment by schauriman** [comment]
OK und vielen Dank, das werde ich mir demnächst ansehen,
In den nächsten Tagen werde dazu sicher nicht kommen aber es wird schon nochmal.
Danke nochmal an alle für eure Hilfe und Bemühungen
